lib/Models/Terria.js

```diff
@@ -363,6 +364,16 @@ Terria.prototype.start = function(options) {
         corsProxy.proxyDomains.push.apply(corsProxy.proxyDomains, config.proxyDomains);
         corsProxy.baseProxyUrl = that.configParameters.corsProxyBaseUrl;
 
+        var promises = [];
+
+        promises.push(loadJson(cp.proxyableDomainsUrl).then(function(proxyableDomains) {
+            if (proxyableDomains.proxyAllDomains) {
+                corsProxy.isOpenProxy = true;
```

Comment:
`proxyAllDomains` is incorrect in this context.  It won't proxy all domains, it's just an indication that the server will allow all domains to be proxied.  It's also not a security problem, the security is in the server.  Setting it to true when the server doesn't agree will just make connections fail.